### PR TITLE
bootstrapper: remove unused CloudServiceAccountUri from init request

### PR DESCRIPTION
**Pull request by elchead**
### Context

Due to the helm chart movement, the parameter is no longer needed.

### Proposed change(s)

-

### Checklist

- [ ] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)
- [ ] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

**Comment by elchead** [comment]
running E2E: https://github.com/edgelesssys/constellation/actions/runs/5797850611

**Comment by elchead** [comment]
will need new nightly images to ensure E2E keeps working

File: bootstrapper/initproto/init.proto [hunk 1]
```diff
@@ -16,8 +16,6 @@ message InitRequest {
   string storage_uri = 2;
   // MeasurementSalt is a salt used to generate the clusterID for the initial bootstrapping node.
   bytes measurement_salt = 3;
-  // CloudServiceAccountUri is an URI encoding the cloud service account.
-  string cloud_service_account_uri = 4;
```
Comment (3u13r) [inline bootstrapper/initproto/init.proto:20]: Until we decide that we want to completely remove the lines of removed entries, I'd prefer:
```
// string cloud_service_account_uri = 4; removed
```

**Review by 3u13r — CHANGES_REQUESTED** [review]

File: bootstrapper/initproto/init.proto [hunk 2]
```diff
@@ -10,14 +10,15 @@ service API {
...
+  reserved "cloud_service_account_uri";
   // KmsUri is an URI encoding access to the KMS service or master secret.
   string kms_uri = 1;
   // StorageUri is an URI encoding access to the storage service.
   string storage_uri = 2;
   // MeasurementSalt is a salt used to generate the clusterID for the initial bootstrapping node.
   bytes measurement_salt = 3;
-  // CloudServiceAccountUri is an URI encoding the cloud service account.
-  string cloud_service_account_uri = 4;
+  // string cloud_service_account_uri = 4; removed
```
Comment (3u13r) [inline bootstrapper/initproto/init.proto:21]: I don't think we should keep this line it if we proceed as stated in their docs, but remove it entirely.

**Review by 3u13r — CHANGES_REQUESTED** [review]

**Review by 3u13r — APPROVED** [review]
LGTM

**Pull request merged**